__tests__/app.test.js

```diff
@@ -23,32 +42,51 @@ describe("/api/topics", () => {
   });
 });
 
-describe("Generic handler", () => {
-  test("GET: 404 sends an appropriate status and error message.", () => {
+describe("/api/articles", () => {
+  test("GET: 200 sends all articles in the correct format.", () => {
     return request(app)
-      .get("/api/top")
-      .expect(404)
+      .get("/api/articles")
+      .expect(200)
       .then((response) => {
-        expect(response.body.msg).toBe("Path not found.");
+        const { articles } = response.body;
+        expect(articles.length).toBe(13);
+        expect(Array.isArray(articles)).toBe(true);
+        articles.forEach((article) => {
+          expect(typeof article).toBe("object");
```

Comment:
We need to be more specific that just checking that the array contains objects. We should be writing an assertion that looks at how the object is shaped(contains the correct keys) and the correct value types for the properties of that object(check the typeof the values)